content/installation/java/Java-linux-packages.md

```diff
@@ -0,0 +1,122 @@
+<!--
+title: "Java Agent Installation via Host Installer"
+description: "Overview of Java agent installation with Host Installer"
+tags: "installation Java agent linux Host Installer"
+-->
+
+## About Installation via System Package Managers
+
```

Comment:
Is 'Host Installer' the official article/feature title that we want to use? Or, should we use something about 'System Package Managers'?

Reply:
The terminology is both confusing and evolving, but it's trending towards better (I think 😐). Here's my understanding of our current terminology:

"Host Installer" refers to the capability we are developing wherein we give users the capability to install one package which configures all future java processes on the host to include the Contrast Java agent. On Linux and Solaris systems, the "Host Installer" capability is fulfilled by installing the `contrast-java-agent-exec-helper` and `contrast-java-agent` packages. On Windows systems, the "Host Installer" capability is fulfilled by installing the "Contrast Java Host Service"; this service works _much_ differently from the Linux and Solaris packages

So where does this leave us? I have been considering keeping "Host Installer" as a term we use internally to describe the effort we are working on, and we omit this term from user facing docs and products so that we do not need to have this conversation with users